### DISCUSSION TITLE
Include "actual size" in error message for content size check

### DISCUSSION THREAD

**Discussion by dylanbussone** [node-fetch/node-fetch#1459] [Ideas]
**Problem:**

The current error message for content size is merely
```
`content size at ${data.url} over limit: ${data.size}`
```
where `data.size` is the limit. It gives no indication of _how much_ bigger the actual size was than the limit, which is valuable information.

**Request:**

Include the actual size in the error message in addition to the limit.
I have a local branch with this addition, if you give me permissions I'd be happy to open a PR.

**Comment by jimmywarting** [comment]
I'm having thoughts on deprecating/removing size option...
https://github.com/node-fetch/node-fetch/issues/1438
